fix_macroeditorsheet_new.py

```
@@ -0,0 +1,168 @@
+                        .contentShape(Rectangle())"""
+
+if search_addstep in content and search_macrosteprow_def in content and search_caller in content:
+    content = content.replace(search_addstep, replace_addstep)
+    content = content.replace(search_macrosteprow_def, replace_macrosteprow_def)
+    content = content.replace(search_caller, replace_caller)
+    with open(filepath, 'w', encoding='utf-8') as f:
+        f.write(content)
+    print("Successfully replaced all blocks without nested buttons.")
+else:
+    print("Search block not found.")
+    if search_addstep not in content: print("addstep failed")
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)